backpack.tf extended sorting.user.js

```diff
@@ -871,6 +874,65 @@ class</a></li>
 
 		}
 
+		var cnSort = 0;
+
+		function sortByCraftNumber() {
+			const sortCN = (a, b) => {
+				try {
+					return Number($(a)[0].outerText.split("#")[1]) - Number($(b)[0].outerText.split("#")[1]);
+				} catch (e) {
+					return parseInt(a[0].split(" ")[1]) - parseInt(b[0].split(" ")[1]);
+				}
```

Comment:
This try catch block is only necessary because you are also supplying this to `genericSort` (see the comment for line 926).
If you do not use this for `genericSort`, you can instead just use:
```suggestion
				return Number($(a)[0].outerText.split("#")[1]) - Number($(b)[0].outerText.split("#")[1]);
```

This is because you already filter for the string including `#` in line 910.